MobiFlight/Board.cs

```diff
@@ -151,6 +184,16 @@ public class Info
         /// </summary>
         public List<String> CustomDeviceTypes = new List<string>();
 
+        /// <summary>
+        /// True if the board ships with a default config that can be auto-loaded after flashing.
+        /// </summary>
+        public Boolean HasDefaultDeviceConfig;
```

Comment:
Does this come from the JSON? Historically we've simply used the presence of a path/value to indicate "has". So in this case, somewhere there would be a `DefaultDeviceConfig` property that either has a path in it or doesn't. It's redundant to specify both the boolean and the path in a .json file.